### DISCUSSION TITLE
Cài đặt ibus-bamboo để gõ Tiếng Việt trên Manjaro

### DISCUSSION THREAD

**Discussion by hungify** [BambooEngine/ibus-bamboo#381] [General]
### 1. Cài đặt IbusBamboo
- `bash -c "$(curl -fsSL https://raw.githubusercontent.com/BambooEngine/ibus-bamboo/master/archlinux/install.sh)"`
### 2.  Cài đặt môi trường
- sudo nano /etc/profile
- gõ các dòng sau đây vào cuối

```
export GTK_IM_MODULE=ibus
export QT_IM_MODULE=ibus
export XMODIFIERS=@im=ibus
export QT4_IM_MODULE=ibus
export CLUTTER_IM_MODULE=ibus
ibus-daemon -drx
```
### 3. Logout sau đó Login và tận hưởng

Mình chỉ làm tới bước 1 thì không hè gõ được tiếng việt
Mình search và có cách này. Nên mình tạo issue này cho bạn nào sau này bị giống mình thử cách này xem sao.

**Comment by zenfas** [comment]
Bước 1 thôi thì không gõ được đúng rồi, phải cài đặt biến môi trường, log out/log in để nó apply

**Comment by hungify** [comment]
Do mình không biết cái dưới á
